README.md

```diff
@@ -1 +1,178 @@
-# toDoList
```

Comment:
| My Logo |
| --- |
|![Capture-modified](https://github.com/Tobby8629/Hello-rails-react/assets/93039359/02370431-b4b5-474a-85bb-0f6fb15de502) |


**[OPTIONAL]**

- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍 Just like I have made my logo which you can see in the image above ⬆️